lib/std/gpu.zig

```diff
@@ -126,3 +127,35 @@ pub fn executionMode(comptime entry_point: anytype, comptime mode: ExecutionMode
         },
     }
 }
+
+/// Writes formatted output to an implementation-defined stream.
+/// Returns 0 on success, –1 on failure.
+pub fn printf(comptime fmt: [*:0]const u8, args: anytype) u32 {
+    if (builtin.zig_backend == .stage2_spirv and builtin.target.os.tag == .opencl) {
```

Comment:
Should `nvcl` be included?

Reply:
Actually it seems `nvcl` is specifically for `nvptx(64)-nvcl`; I don't think `spirv64-nvcl` is a thing. And there's no `nvptx(64)-opencl`. So the answer to my question seems to be no.

But we should probably delete `nvcl` and lower `nvptx(64)-opencl` to `nvptx(64)-nvcl` when talking to LLVM.